components/authz-service/server/v2/project.go

```diff
@@ -155,23 +155,37 @@ func (s *state) UpdateProject(ctx context.Context,
 			"error converting project with ID %q: %s", resp.ID, err.Error())
 	}
 
-	err = s.projectUpdateManager.Start()
+	return &api.UpdateProjectResp{Project: apiProject}, nil
+}
+
+func (s *state) ApplyRulesStart(ctx context.Context, req *api.ApplyRulesStartReq) (*api.ApplyRulesStartResp, error) {
```

Comment:
This doesn't use `req`, let's replace it, **and more importantly** it doesn't use `ctx`. Is this something to fix in the `projectUpdateManager` API?
```suggestion
func (s *state) ApplyRulesStart(context.Context, *api.ApplyRulesStartReq) (*api.ApplyRulesStartResp, error) {
```

Looks like the same holds for the other methods, re: `ctx` not being used.